### DISCUSSION TITLE
Error when trying to install homeassistant with 24.04 Ubuntu

### DISCUSSION THREAD

**Discussion by ggoosen** [tteck/Proxmox#3472] [General] [answered]
sing Advanced Settings
Using Distribution: ubuntu
Using ubuntu Version: 24.04
Using Container Type: 0
Using Root Password: ********
Container ID: 100
Using Hostname: homeassistant.local
Using Disk Size: 20
Allocated Cores: 2
Allocated RAM: 2048
Using Bridge: vmbr0
Using IP Address: 192.168.0.100/24
Using Gateway IP Address: 192.168.0.1
Using APT-Cacher IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using Vlan: Default
Enable Root SSH Access: yes
Enable Verbose Mode: no
Creating a Home Assistant-Core LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using ssd_data for Container Storage.
 ✓ Updated LXC Template List
|unable to create CT 100 - unsupported Ubuntu version '24.04'
bash: line 182: exit: A problem occured while trying to create container.: numeric argument required

**Comment by tteck** [comment, marked as answer]
![Screenshot 2024-08-05 3 25 02 AM](https://github.com/user-attachments/assets/15063d7b-b230-44cd-a9e4-6814d8839844)
